content/en/docs/Installation/trustedboot.md

```diff
@@ -121,6 +121,44 @@ title My Awesome OS
 Remember when you build upgrade mediums, to also add your branding, otherwise the new active image will be named differently than your fallback and recovery ones.
 {{% /alert %}}
 
+### Additional efi entries
+
+When booting in UKI mode, the cmdline is part of the signed artifact. This means that in order to pass additional options through the cmdline,
+an additional efi entry is needed. `enki build-uki` supports the following arguments:
+
+- `--extend-cmdline`: Use this option to add options to the cmdline used for active/passive/recovery. This option will not generate additional entries in the systemd-boot menu.
+- `--extra-cmdline`: Use this option to generate additional entries in the systemd-boot menu. For each for active/passive/recovery, a new entry will be created with the specified cmdline (added to the default cmdline).
+- `--single-efi-cmdline`: Use this option to generate entries that are not related to active/passive/recovery. This option will generate a single entry with the provided cmdline (added to the default cmdline). This argument can be used multiple times.
+
+
+Examples:
+
+Building with a command like this:
+
+```
+enki build-uki ...more options... --single-efi-cmdline "Lets debug: rd.debug rd.immucore.debug"
+```
+
+Will create a boot menu with the default entries, plus one more: "Lets debug" which will have the debug options set.
```

Comment:
yeah here we use "default" entries and sounds good, so I think it would be safe to call it Kairos default entries